docs/source/index.rst

```diff
@@ -1,81 +1,51 @@
 |PyPI| |Bioconda| |Downloads| |CI| |Notebooks| |Docs| |Codecov|
 
-CellRank for directed single-cell fate mapping
-==============================================
+CellRank: dynamics from multi-view single-cell data
+===================================================
 
-.. image:: https://raw.githubusercontent.com/theislab/cellrank/master/resources/images/cellrank_fate_map.png
+.. image:: https://raw.githubusercontent.com/theislab/cellrank/master/resources/images/cellrank_overview.png
```

Comment:
Could we move this to `docs/source/_static/img`?

Reply:
done.